ucsb-cs156-f23/proj-happycows-f23-5pm-3
# #10 Changed commons default values to be settable by .env variables a…

…long with associated tests

## Overview

In this PR, I changed the default values of the commons to be settable through environment variables. I also created a new endpoint /default which allows a user to query for a default commons object.

## Feedback Request 

Please give suggestions on the following:
- Is this the best way to use application.properties?

## Future Possibilities (Optional)

Some possible points could be:
- Default values for all members of Commons objects for consistencies sake unless this is unwanted

## Validation (Optional)

1. Pull this branch
2. Start up the application on localhost
3. Got to swagger-ui 
4. Execute the /default endpoint and see if you get the fallback values in application.properties
5. Then set environment variables and see if you get those when you execute /default

## Tests


## Linked Issues

Closes #16
